src/Select/Select.jsx

```diff
@@ -44,6 +44,22 @@ export function Select({
 }) {
   const {cssClass} = Select;
 
+  if (!lazy) {
```

Comment:
Hmm I find that schema pretty confusing (array or object with lazy attribute set to true and a loadOptions function). I'll change to console.warn, but I'm going to keep it as is.